charts/images.yaml

```
@@ -32,6 +32,10 @@ images:
   sourceRepository: github.com/gardener/machine-controller-manager
   repository: eu.gcr.io/gardener-project/gardener/machine-controller-manager
   tag: "v0.37.0"
+- name: machine-controller-manager-provider-openstack
+  sourceRepository: github.com/gardener/machine-controller-manager-provider-openstack
+  repository: eu.gcr.io/gardener-project/gardener/machine-controller-manager-provider-openstack
+  tag: "v0.0.1"
```

I thought this release version should have been, however, I see your release version matching `v0.0.1`.  You can take a final call on this. 
```suggestion
  tag: "v0.1.0"
```

---

The idea was to release `v0.0.1` as an alpha version to validate on dev and have the leeway to release `v0.1.0` if there is a need to do fixes after validation.

---

Okay sounds good. Thanks.